styles.go

```diff
@@ -1885,13 +1885,9 @@ func parseFormatStyleSet(style string) (*formatStyle, error) {
 //
 // Cell Sheet1!A6 in the Excel Application: martes, 04 de Julio de 2017
 //
-func (f *File) NewStyle(style string) (int, error) {
+func (f *File) NewStyle(fs *FormatStyle) (int, error) {
```

Comment:
I think you should add a new function that takes the newly exposed style struct instead of changing the signature of NewStyle.

As it stands, we would have to release v3 of the library since you are making a breaking change to how people would consume it. If you add a new function to take the struct like `NewStyleStruct` or `AddStyle` (or something...) then this can go out as v2.1.0.

All the private functions you changed seem fine since consumers can't see those.